tests/radv/test_radv_restart.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
+import pytest
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,  # disable automatic loganalyzer globally
+    pytest.mark.topology('any')
+]
+
+
+# For this Test gap "Restarting radv causes swss service to restart"
+# see details at
+# https://github.com/sonic-net/sonic-mgmt/issues/6042
+def test_radv_swss(duthost):
+    """swss does not restart while radv restarting
+
+    Args:
+        duthost: AnsiblecHost instance for DUT
+    """
+
+    # 1. check status of swss.service
+    swss_stdout_lines = duthost.shell("systemctl show -p ActiveState -p ActiveEnterTimestamp swss.service")[
+        "stdout_lines"]
+    swss_status_dict_before = parse_service_status(swss_stdout_lines)
+
+    # make sure swss is running
+    assert swss_status_dict_before is not None and swss_status_dict_before.get("ActiveState") == "active"
```

Comment:
If supply a default value to the `get` method, the code would be more robust. It won't hit `KeyError`. Similar for the other `get` calls.
The other suggestion is to add an error message for the assertion. The message will be captured by pytest as a summary for this case if it failed. It would be useful for analyzing test results.

Reply:
I learned that `get`method has a default value `None`, So that the origin code is `get("ActiveState",None)`, and it can be simplified as `get("ActiveState")`.
For your suggestion about error message for the assertion, I'll be think about it and change later. Thanks.